src/components-examples/material/sidenav/sidenav-responsive/sidenav-responsive-example.html

```diff
@@ -1,25 +1,21 @@
 @if (shouldRun) {
-  <div class="example-container" [class.example-is-mobile]="mobileQuery.matches">
+  <div class="example-container" [class.example-is-mobile]="mobileQuery">
```

Comment:
We can clean this up even more with the `async` pipe instead of having to subscribe manually. Here's an example: https://github.com/angular/components/blob/main/src/dev-app/screen-type/screen-type-demo.html#L4